src/components/TrackTime/helpers.ts

```diff
@@ -35,3 +44,38 @@ export function customizedTime(time: FormattedTime, locale: Locale): string {
 
   return `${formattedHours}${hours} ${formattedMinutes}${minutes} ${formattedSeconds}${seconds}`
 }
+
+export function customizedPeriod(
+  dates: TimePeriod[],
+  language: Language,
+): string {
+  if (!dates.length) return ''
+
+  moment.locale(language)
```

Comment:
uses moment.locale(language) to set the language globally this could have side effects.
 Consider scoping this setting locally (e.g., `moment(startDate).locale(language).format(...))` to avoid unwanted side effects.